guides/common/attributes-titles.adoc

```diff
@@ -24,6 +24,7 @@
 :MonitoringDocTitle: Monitoring {Project} performance
 :PlanningDocTitle: Planning for {ProjectName}
 :ProvisioningDocTitle: Provisioning hosts
+:ProvisioningNetbootDocTitle: Provisioning machines by using network boot
```

Comment:
I wasn't sure about the title of this guide:

1. "Provisioning machines by using network boot" is not entirely precise - this guide covers also boot disk booting, which is a utility that supplies network booting when the firmware of a host doesn't.
2. "Provisioning bare-metal machines" is not precise either because the procedures are not limited by the type of the host (bare metal vs virtual) but it is limited by how much control the Foreman admin has over the network services, especially DHCP.
3. "Provisioning machines by using an installer" - This probably covers it better than the current title.

Please leave me :+1: if you agree with option 3. or let me know in a comment if you have another idea.

Reply:
This might sound heretical  😆 but I am not a big fan of beginning guide names with gerunds. It forces the user to read long titles to the very end in order to figure out whether it's the guide they want: 

- Installing our product with 300 features and bells and whistles and notifications by using PXE boot.
- Installing our product with 300 features and bells and whistles and notifications by using ISO images.

In the old days, we would just call the guide something like "PXE-boot installation guide" or "ISO image installation guide". However, those days are gone. Our guides have to sound like everyone else's, at least as far as d/s docs are concerned.

I want to take a closer look at the options you are proposing before I cast my vote. :smile:

Reply:
I looked at your Miro board and Jira. Option 3 seems to be OK, even with the caveat about PXE mode. Any reason why you changed "Provisioning hosts" to "Provisioning machines"? 

If you change the terminology, you might have to update section headers so that they are consistent.

Reply:
I've decided to go with "Provisioning hosts by using installer automation" to be super specific.